sdk/translation/ai-translation-text-rest/tsp-location.yaml

```diff
@@ -1,3 +1,3 @@
 directory: specification/translation/Azure.AI.TextTranslation
-commit: 01a71545e82bb98b8137d3038150c436d46a59ed
-repo: Azure/azure-rest-api-specs
+commit: 3af1f0b4582e775d1dfefe578781671fc599296f
+repo: MikeyMCZ/azure-rest-api-specs
```

Comment:
why in private repo? not azure one?

Reply:
TypeSpec PR is still pending - https://github.com/Azure/azure-rest-api-specs/pull/28920

Reply:
Fixed